src/BasketEmail.js

```diff
@@ -0,0 +1,239 @@
+
+import React from 'react';
+import './BasketEmail.scss';
+import PropTypes from 'prop-types';
+import './BasketSend.scss';
+
+class BasketEmail extends React.Component {
+  state = {
+    mapOfInform: {
+      name: '',
+      email: '',
+      phone: '',
+      country: '',
+    },
+    errorsMap: {
+      name: '',
+      email: '',
+      phone: '',
+      country: '',
+    },
+  }
+
+  handleInputFocus = (event) => {
+    const errorClear = event.target.name;
```

Comment:
если это имя инпута, почему `errorClear`
```suggestion
    const { name } = target;
```